src/components/ha-attributes.ts

```diff
@@ -0,0 +1,97 @@
+import {
+  property,
+  LitElement,
+  TemplateResult,
+  html,
+  CSSResult,
+  css,
+  customElement,
+} from "lit-element";
+import { HassEntity } from "home-assistant-js-websocket";
+import { repeat } from "lit-html/directives/repeat";
+
+import hassAttributeUtil from "../util/hass-attributes-util";
+
+@customElement("ha-attributes")
+class HaAttributes extends LitElement {
+  @property() public stateObj?: HassEntity;
+  @property() public extraFilters?: string;
+
+  protected render(): TemplateResult | void {
+    if (!this.stateObj) {
+      return html``;
+    }
+
+    const items = this.computeDisplayAttributes(
+      Object.keys(hassAttributeUtil.LOGIC_STATE_ATTRIBUTES).concat(
+        this.extraFilters ? this.extraFilters.split(",") : []
+      )
+    );
+
+    return html`
+      <div class="layout vertical">
+        ${repeat(
+          items,
+          (attribute) => attribute,
+          (attribute) => html`
+            <div class="data-entry layout justified horizontal">
```

Comment:
classes `layout justified horizontal` are not defined, are they not needed or missing?

Reply:
I think they are from the old included styles? maybe? not sure. Will remove.